app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,46 @@
+class TasksController < ApplicationController
+
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+    @task = Task.find(params[:id])
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.new(task_params)
+    if @task.save!
```

Comment:
Noticed you put a `!` on your `save` here. This would return an exception if `save` failed. However, this would be bad for your controller, as your controller only handles `truthy`/`falsey` values (see if..else statements). 

See https://stackoverflow.com/questions/1761076/when-do-i-use-save-create-and-update-attributes-in-rails
and https://robots.thoughtbot.com/save-bang-your-head-active-record-will-drive-you-mad for a further exploration into the matter :)

Reply:
Okay wil hge a read, thanks!